docs/zh_CN/xparl/serialize.md

```diff
@@ -0,0 +1,22 @@
+# **序列化/反序列化加速**
+
+PARL默认使用cloudpickle库进行数据的序列化和反序列化【数据是以序列化后的字节流形式在xparl中进行传输】；如果Python环境有下载pyarrow库的话，则会使用pyarrow库进行序列化和反序列化（由于pyarrow兼容性不够好，PARL不会默认下载该库）。
+
+不同数据场景下，pyarrow和cloudpickle的表现优劣不同，用户可以基于自己的使用场景选择是否要下载pyarrow库。
+
+### 性能对比
+这里提供了pyarrow和cloudpickle在不同数据下的序列化和反序列化的平均耗时作为参考：
+
+- 测试数据一：`data = [np.random.RandomState(0).randn(50, 50)] * 10`
+- 测试数据二：`data = [np.random.RandomState(0).randn(500, 500)] * 10`
+- 测试数据三：`data = [np.random.RandomState(0).randn(5000, 5000)] * 10`
+- 测试数据四：`data = np.random.RandomState(0).randn(5000, 50000)`
+
+> pyarrow版本：python2使用pyarrow==0.16.0，python3中使用pyarrow==2.0.0
+
+<img src="./.images/dataset1.png" width="400"/> <img src="./.images/dataset2.png" width="400"/>
```

Comment:
The title of images should be written in English. We have to rewrite the English documentation in the near term.

Reply:
done